scripts/update/v8_12_0/AddIndexesToDeliveryLogTable.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+namespace oat\taoProctoring\scripts\update\v8_12_0;
+
+
+use oat\oatbox\action\Action;
+use oat\oatbox\service\ServiceManager;
+use oat\taoProctoring\model\deliveryLog\implementation\RdsDeliveryLogService;
+
+class AddIndexesToDeliveryLogTable implements Action
```

Comment:
could u implements `AbstractAction` and use `$this->getServiceLocator()` (or better Script Action to provide description)